libs/perun/dialogs/src/lib/edit-facility-resource-group-vo-dialog/edit-facility-resource-group-vo-dialog.component.html

```diff
@@ -19,11 +19,7 @@ <h1 *ngIf="dialogType === 2" mat-dialog-title>{{'DIALOGS.EDIT_FACILITY_RESOURCE_
       <mat-form-field>
         <input [formControl]="nameCtrl"
                matInput
-               placeholder="{{'DIALOGS.EDIT_FACILITY_RESOURCE_GROUP_VO.NAME' | translate}}:"
-               required>
-        <mat-error>
-          {{invalidNameMessage.length ? invalidNameMessage : ('DIALOGS.EDIT_FACILITY_RESOURCE_GROUP_VO.FILL_NAME' | translate)}}
-        </mat-error>
+               placeholder="{{'DIALOGS.EDIT_FACILITY_RESOURCE_GROUP_VO.NAME' | translate}}:">
```

Comment:
I think that you removed comletely wrong code, in edit dialogs the description field stil have constriction
![image](https://user-images.githubusercontent.com/32871167/134301007-1b500838-a35d-4edf-9008-505d6e80f882.png)